README.md

```diff
@@ -137,10 +141,43 @@ e.g.:
 {
   "oip": {
     "color": {
-      "fg-muted": {"value": "#000000"}
+      "fg-muted": {
+        "value": "#000000"
+      }
     }
   }
 }
 ```
 
 The latter form is harder to keep track off across files though.
+
+### Add as a submodule instead of NPM
+
+In the root folder of your project, create a dir or directly add the submodule in its own directory:
```

Comment:
`create a dir or directly add the submodule in its own directory`: unnecesary, the command is self-explanatory